web/src/components/sidebars/LayersPanel.vue

```diff
@@ -34,14 +34,20 @@ function setVisibility(layers: Layer[], visible=true) {
 const debouncedUpdateFrame = debounce((layer: Layer, value: number) => {
     layerStore.selectedLayers = layerStore.selectedLayers.map((l: Layer) => {
         if (l.id === layer.id && l.copy_id === layer.copy_id) {
-            l.current_frame = value;
+            l.current_frame_index = value;
         }
         return l;
     })
 }, 10)
 
 function getLayerMaxFrames(layer: Layer) {
-    return [...new Set(layer.frames.map((f) => f.index))].length
+    return [...new Set(
+        layerStore.layerFrames(layer).map((f) => f.index)
+    )].length
```

Comment:
I know this existed prior to this PR, but I just noticed this, and since it's being modified anyway, the array expansion is unnecessary:
```suggestion
    return new Set(
        layerStore.layerFrames(layer).map((f) => f.index)
    ).size
```